src/drivers/apl-ssp.c

```diff
@@ -579,16 +578,16 @@ static inline int ssp_set_config(struct dai *dai,
 	ssp_write(dai, SSTSA, sstsa);
 	ssp_write(dai, SSRSA, ssrsa);
 
-	trace_value(sscr0);
```

Comment:
no such logic. It is enabled at the end of sof_probe and before topology loading so that we can check the topology loading

Reply:
ok, so I would suggest to schedule a DMA copy immediately after the DMA host buffers are passed via IPC. This will offload all boot trace to the host freeing up space for topology.
I would also make sure that the driver sends host trace buffer data IPC  before sending any topology too.

Reply:
Ok, i will try this.

Reply:
I will refine it after  1.2 release.